frontend/src/metabase/dashboard/components/QuestionPickerModal/QuestionPickerModal.unit.spec.tsx

```diff
@@ -85,19 +85,10 @@ async function setup() {
 }
 
 describe("QuestionPickerModal", () => {
-  beforeEach(() => {
```

Comment:
Fake timers we used here to speed up a debounced search. I wasn't able to find a way to use fake timers while also using `waitForLoaderToBeRemoved`